website/docs/gs/dockerfiles.md

```diff
@@ -0,0 +1,77 @@
+---
+title: Docker Images
+description: Building and hosting of the Docker Images
+sidebar_position: 4
+---
+
+The tools needed to run the GATK-SV pipeline are organized in 
+different Docker images. These images are automatically 
+built in GATK-SV [_Continuous Integration and Continuous Delivery_ (CI/CD)](https://github.com/broadinstitute/gatk-sv/tree/main/.github/workflows)
+and hosted on Google Container Registry (GCR). 
+The [`dockers.json`](https://github.com/broadinstitute/gatk-sv/blob/main/inputs/values/dockers.json)
+file contains an up-to-date list of Docker images.
+
+
+:::info
+The process of building Docker images, uploading to a container registry, and updating the `docker.json`
+file is completely automated. You do not need to re-run this process unless for development purposes.
+
+The GATK-SV CI/CD does not update the references to Docker images in your workflows. 
+Therefore, you need to consult with `dockers.json` for the latest current Docker images to be used 
+in your workflows. 
+:::
+
+### When should a Docker image be updated?
```

Comment:
I think that to a user clicking through the website it is somewhat unclear that they are not responsible for docker images - it is noted in this document, but organizing it with "Getting started" feels confusing. I would suggest creating a "For developers" section and putting this there instead of in "Getting started", or even just creating a README for the `/scripts/docker/` directory of the repo

Reply:
I remove this file, we can revisit and add a section that describes the Docker file and their build process since they are one of the inputs of the pipeline.